Add upgrading steps (#13)
RockefellerArchiveCenter/archivematica-local: appendix.md

*  If PREMIS _act_ = Disseminate and PREMIS _restriction_ = Conditional, Publish = FALSE
*  If PREMIS _act_ = Disseminate and PREMIS _restriction_ = Disallow, Publish = FALSE

## FPR Customatizations

The following customizations have been added to the Format Policy Registry (the "Preservation Planning" tab):

### Characterization Rules

> *  ffprobe and MediaInfo should be disables for JPG and TIFF files, as ExifTool is sufficient for characterizing these formats

Reviewer: ```suggestion
*  ffprobe and MediaInfo should be disabled for JPG and TIFF files, as ExifTool is sufficient for characterizing these formats
```

## ## Upgrading to a new release of Archivematica

### 1. Review Release Notes (Archivematica Product Owner)

Review release notes made available on Archivematica wiki, noting any changes that may impact RAC workflows or the upgrade process. If skipping versions (e.g., from 1.7 to 1.9), review release notes for intermediate versions.

### 2. Plan Upgrade (Archivematica Product Owner)

> Coordinate with Artefactual, Information Systems Manager, and key RAC staff [link] to plan upgrade at least 3 weeks in advance. Production and development upgrade should be planned at the same time, with production scheduled for 2 weeks after development. The upgrade schedule will likely look like:

Reviewer: ```suggestion
Coordinate with Artefactual, RAC Information Systems Manager, and other key RAC staff [link] to plan upgrade at least 3 weeks in advance. Production and development upgrade should be planned at the same time, with production scheduled for 2 weeks after development. The upgrade schedule will likely look like:
```
Is this missing a link?

* Week 1: Set up development VMs, upgrade development
* Week 2: Upgrade integrated applications, test development
* Week 3: Set up production VMs, upgrade production
* Week 4: Upgrade integrated applications, test production

### 3. Set up Development VMs (Information Systems Manager) (optional)

> This is only necessary if we are upgrading Archivematica on new VMs. This step includes:

Reviewer: ```suggestion
This is only necessary if the Archivematica upgrade involves a move to new VMs. This step includes:
```

* Setting up ssh access for [select RAC staff](https://docs.rockarch.org/systems-info-sheets/archivematica-info-sheet/) for VMs
* Setting up all accounts and access for Artefactual
* Configuring [DNS](https://docs.rockarch.org/systems-info-sheets/archivematica-info-sheet/

When this is complete, convey information about IPs and logins to the Archivematica Product Owner so that she can update the Archivematica Info Sheet, inform key staff, and let Artefactual know our development environment is ready to be upgraded.

### 4. Upgrade (Artefactual)

Archivematica Product Owner alerts key staff to window where Archviematica will be down. Artefactual upgrades both pipelines and the storage service in our dev environment.

### 5. Update Integrated Applications (Archivematica Product Owner)

Review whether user accounts, API Keys, and Pipeline and Location UUIDs have changed, and work with key staff to update configs in other applications as necessary. Connected applications may include Fornax and Gemini.

### 6. Test on Development (Archivematica Product Owner)

Along with key staff, test the following:

> * All expected user accounts [link] are present

Reviewer: missing link

* Setting up ssh access for [select RAC staff](https://docs.rockarch.org/systems-info-sheets/archivematica-info-sheet/) for VMs
* Setting up all accounts and access for Artefactual
* Configuring [DNS](https://docs.rockarch.org/systems-info-sheets/archivematica-info-sheet/

### 8. Upgrade (Artefactual)

Archivematica Product Owner alerts key staff to window where Archviematica will be down. Artefactual upgrades both pipelines and the storage service in our production environment.

### 9. Update Integrated Applications (Archivematica Product Owner)

Review whether user accounts, API Keys, and Pipeline and Location UUIds have changed, and work with key staff to update configs in other applications as necessary. Connected applications may include Fornax and Gemini.

### 10. Test on Production (Archivematica Product Owner)

Along with key staff, test the following:

* All expected [user accounts](https://docs.rockarch.org/systems-info-sheets/archivematica-info-sheet/) are present
* Transfers can be sent to and stored in all locations on both pipelines
* All integrations work as expected

> Errors as they occur will be communicated to the Archivematica Product Owner, who will work with Artefactual to resolve them. When testing is complete, the Archivematica Product Owner will alert Artefactual and key staff.

Reviewer: ```suggestion
Errors will be reported to the Archivematica Product Owner, who will work with Artefactual to resolve them. When testing is complete, the Archivematica Product Owner will alert Artefactual and key staff.
```
